lib/foreman/renderer/scope/macros/helpers.rb

```diff
@@ -11,6 +11,31 @@ module Helpers
             sections only: %w[all]
           end
 
+          apipie :method, "Returns next integer stored in a hostgroup param named 'sequence_num', starts and increments it by a given amount" do
+            required :hostgroup_name, String, "hostgroup title to store in (e.g. RHEL7/base)"
+            optional :increment, Integer, "increment (defaults to 1)"
+            optional :name, String, "name of the sequence ('default' when not provided)"
```

Comment:
you can specify default value for `optional` params as an argument:

`optional :increment, Integer, "value to increment by", default: 1`
`optional :name, String, "name of the sequence", default: 'default'`

Reply:
Thanks, I am also adding leading zeroes param.